frontend/src/theme/custom.scss

```diff
@@ -124,3 +124,25 @@ ion-item.item-has-focus {
   flex-grow: 0;
   padding-top: 1rem;
 }
+
+/*
+  Customization of the Popover in the dropdownmenu of the field-designation.page
+*/
+.custom-popover {
+  [aria-checked='true'].sc-ion-alert-ios .alert-checkbox-icon.sc-ion-alert-ios,
+  [aria-checked='true'].sc-ion-alert-md .alert-checkbox-icon.sc-ion-alert-md {
+    border-color: var(--ion-color-tertiary);
+    background-color: var(--ion-color-contrast);
+  }
+
+  .alert-checkbox-label.sc-ion-alert-ios,
+  .alert-checkbox-label.sc-ion-alert-md {
+    //font-size: 18px;
```

Comment:
Maybe, remove that line.

Reply:
Okay, that was rest of some testing the designs, but i didn't liked it.